openapi/openapi/src/main/scala/endpoints4s/openapi/ChunkedEntities.scala

```diff
@@ -40,14 +40,27 @@ trait ChunkedJsonEntities
   private def jsonChunksEntity[A](codec: JsonCodec[A]) =
     Map("application/json" -> MediaType(Some(toSchema(codec.docs))))
```

Comment:
I think we should document the framing strategy as well.

I don’t remember what is the level of support in OpenAPI to do so, though.

Reply:
I am not too experienced with OpenAPI, but there still is an open PR here for this https://github.com/OAI/OpenAPI-Specification/issues/1576. So we do have to come up with something of our own. Should we just add some text to the description field?

Also, `application/json` is an incorrect content type for `ChunkedJsonEntities`, because the request/response bodies are not valid JSON documents.

Reply:
Thank you for the link!

I agree that `application/json` is wrong (unless we used a framing strategy that wraps the whole stream into a JSON array, where frames would be items of the array). I am not sure what we should use instead?

It seems the support for streaming in OpenAPI is very poor, so maybe there is very little we can do about it…

Reply:
Hmm, maybe we could express the content type's dependency on framing then? For example, `newLineDelimiterFraming` could set it to something like `text/plain`, but if someone would implement the JSON array framing, then it could set it to `application/json`?

Reply:
I suggest revisiting this later if someone actually implements a JSON array framing strategy.